bom/pom.xml

```diff
@@ -116,12 +116,12 @@
         <json-smart.version>2.3</json-smart.version>
         <infinispan.version>10.0.0.Beta3</infinispan.version>
         <caffeine.version>2.6.2</caffeine.version>
-        <netty.version>4.1.34.Final</netty.version>
+        <netty.version>4.1.36.Final</netty.version>
```

Comment:
Again we do not want to upgrade Netty: it should be coordinated across all components.